build/Versions.props

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Properties are commented out where they match the current default specified in Repo Toolset -->
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <!-- Repo Version Information -->
+  <PropertyGroup>
+    <VersionBase>2.0.3</VersionBase>
+    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
+  </PropertyGroup>
+
+  <!-- Repo Toolset Features -->
+  <PropertyGroup>
+    <!-- <UsingToolIbcOptimization>false</UsingToolIbcOptimization> -->
+    <UsingToolMicrosoftNetCompilers>false</UsingToolMicrosoftNetCompilers>
+    <!-- <UsingToolPdbConverter>false</UsingToolPdbConverter> -->
+    <UsingToolSourceLink>false</UsingToolSourceLink>
+    <UsingToolVSSDK>true</UsingToolVSSDK>
+    <!-- <UsingToolXliff>true</UsingToolXliff> -->
+    <!-- <UsingToolXUnit>true</UsingToolXUnit> -->
+  </PropertyGroup>
+
+  <!-- Toolset Dependencies -->
+  <PropertyGroup>
+    <DotNetCliVersion>2.0.2-vspre-006963</DotNetCliVersion>
+    <!-- <MicroBuildCoreVersion>0.2.0</MicroBuildCoreVersion> -->
+    <!-- <MicroBuildPluginsSwixBuildVersion>1.0.147</MicroBuildPluginsSwixBuildVersion> -->
+    <!-- <MicrosoftDotNetIBCMergeVersion>4.7.1-alpha-00001</MicrosoftDotNetIBCMergeVersion> -->
+    <!-- <MicrosoftNetCompilersVersion>2.3.0</MicrosoftNetCompilersVersion> -->
+    <!-- <Pdb2PdbVersion>1.1.0-beta1-62209-01</Pdb2PdbVersion> -->
+    <!-- <RoslynToolsMicrosoftModifyVsixManifestVersion>0.3.3-beta</RoslynToolsMicrosoftModifyVsixManifestVersion> -->
+    <RoslynToolsMicrosoftRepoToolsetVersion>1.0.0-alpha45</RoslynToolsMicrosoftRepoToolsetVersion>
+    <RoslynToolsMicrosoftSignToolVersion>0.3.4-beta</RoslynToolsMicrosoftSignToolVersion>
+    <SourceLinkCreateCommandLineVersion>2.5.0</SourceLinkCreateCommandLineVersion>
+    <VSWhereVersion>2.2.7</VSWhereVersion>
+    <!-- <XliffTasksVersion>0.2.0-beta-000053</XliffTasksVersion> -->
```

Comment:
Why is this commented out?

Reply:
See the comment at the top of the file: https://github.com/dotnet/sdk/pull/1702/files/fb83bc7ce80f5f12ad21bf408678ee1f000aa331#diff-9260067da6a2db86b9a5db30fa8ee2aeR5